packages/react/src/primitives/Alert/Alert.tsx

```diff
@@ -51,22 +51,19 @@ export const Alert: Primitive<AlertProps, typeof Flex> = ({
         wrap={wrap}
         {...rest}
       >
-        <Flex alignItems="center">
-          {hasIcon && <AlertIcon variation={variation} iconSize={iconSize} />}
-          <View role="alert">
-            {heading && (
-              <Heading
-                className={ComponentClassNames.AlertHeading}
-                level={headingLevel}
-              >
-                {heading}
-              </Heading>
-            )}
-            {children}
-          </View>
-        </Flex>
+        {hasIcon && <AlertIcon variation={variation} />}
+        <View role="alert" flex="1">
+          {heading && (
+            <View className={ComponentClassNames.AlertHeading}>{heading}</View>
```

Comment:
heading is now just a `View` with a className because semantically it does not need a specific heading level. An alert can appear anywhere in the document.